quickwit-storage/src/bundle_storage.rs

```diff
@@ -134,6 +134,14 @@ impl BundleStorageFileOffsets {
 
 #[async_trait]
 impl Storage for BundleStorage {
+    async fn check(&self) -> anyhow::Result<()> {
+        if self.bundle_filepath.exists() {
+            Ok(())
+        } else {
+            anyhow::bail!("Missing path `{}`", self.bundle_filepath.display())
+        }
```

Comment:
Super Nitpick:

We tend to prefer early return in the codebase. This is really 
a matter of convention and you could not know of course :).
```
if !self.bundle_filepath.exists() {
    anyhow::bail!("Missing path `{}`", self.bundle_filepath.display())
}
Ok(())
```